### ISSUE TITLE
[sub-issue] Contrast Problem in Symbols

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#325]
Not enough contrasts:

![image](https://cloud.githubusercontent.com/assets/7510842/5357864/8db103dc-7fad-11e4-8c16-57cd67348c24.png)
Main Navigation
==> Use #454545

Other symbols with insufficient contrast:

![image](https://cloud.githubusercontent.com/assets/7510842/5357971/cda19884-7fae-11e4-818d-d0592345d60f.png)
Dropdown
==> Use #454545

![image](https://cloud.githubusercontent.com/assets/7510842/5358033/b4dabe2e-7faf-11e4-9980-69c42bd6a4a1.png)
Carousel
==> Use #454545

![image](https://cloud.githubusercontent.com/assets/7510842/5358007/4f258032-7faf-11e4-942a-83a24a2851bc.png)
Flyout
==> Use #454545
